charts/pega/values.yaml

```diff
@@ -162,12 +162,16 @@ global:
             serverName: ""
             # set insecureSkipVerify=true, if the certificate verification has to be skipped
             insecureSkipVerify: false
-        serviceType: "LoadBalancer"
-        # Specify the CIDR range inside the qoutations to restrict the service access to the given CIDR range
-        # e.g "123.123.123.0/24"
-        loadBalancerSourceRanges:
-          - "103.38.88.0/24"
-          - "49.43.219.90/32"
+        # Use this parameter to deploy service of a specifictype. This is an optional value and should be used based on the usecase.
+        # serviceType: ""
+        # Should be used only when serviceType is set to LoadBalancer.
```

Comment:
use spacing between words. If servicetype requires, LoadBalancer without a space, you have to call that out.

Is this the context? I'm guessing here...please add any missing details>
To specify a load balancing service, uncomment the previous line to enable its use and specify, `serviceType:` "LoadBalancer".

Reply:
Made changes according to the comments.